packages/teuchos/comm/src/Teuchos_StackedTimer.hpp

```diff
@@ -653,6 +656,7 @@ class StackedTimer
       total_updates_(0),
       min_(0),
       max_(0),
+      procmax_(0),
```

Comment:
Again, prefer member initializers to implementing a nondefault default constructor.